README.md

```diff
@@ -75,7 +75,7 @@ Build your container image with:
 mvn compile jib:build
 ```
 
-Subsequent builds are much faster than the initial build.
+Subsequent builds are much faster than the initial build. `mvn clean` cleans Jib's build cache and forces Jib to re-pull the base image and re-build the application layers.
```

Comment:
This new sentence seems disconnected from the rest of the section. 
Are you trying to say that `mvn compile jib:build` should be the command you run to build your image and not `mvn clean compile jib:build`?

Reply:
Yes, the idea is that `mvn compile jib:build` is the command to run in most cases, and if you want to clear Jib's cache, `mvn clean` would do that.

Reply:
Ok, what if you worded it like this:

Build your container image with:
 
 ```commandline
 mvn compile jib:build
 ```
Subsequent builds are much faster than the initial build. 

If you want to clear Jib's build cache and force it to re-pull the base image and re-build the application layers, run:

 ```commandline
 mvn clean compile jib:build
 ```

Reply:
That sounds much clearer! Will update with this